.github/workflows/issue-labeler.yml

```
@@ -0,0 +1,35 @@
+name: Issue labeler
+on:
+  issues:
+    types: [ opened ]
+
+permissions:
+  contents: read
+
+jobs:
+  label-component:
+    runs-on: ubuntu-latest
+
+    permissions:
+      # required for all workflows
+      issues: write
```

Don't think "all workflows" is correct:

```suggestion
      # required to set labels on issues
      issues: write
```

---

updated